sqlglot/dialects/spark.py

```diff
@@ -1,223 +1,53 @@
-from __future__ import annotations
```

Comment:
why did you remove this

Reply:
I think just copy-paste mistake, I will add it back again.